examples/flutter_firebaseauth/lib/main.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:flutter/material.dart';
+
+import 'package:bloc/bloc.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_firebaseauth/authentication/authentication.dart';
+import 'package:flutter_firebaseauth/authentication/authentication_service.dart';
+import 'package:flutter_firebaseauth/home_page.dart';
+import 'package:flutter_firebaseauth/login_page.dart';
+
+class SimpleBlocDelegate extends BlocDelegate {
+  @override
+  void onTransition(Transition transition) {
+    print(transition);
+  }
+
+  @override
+  void onError(Object error, StackTrace stacktrace) {
+    print(error);
+  }
+}
+
+void main() {
+  BlocSupervisor().delegate = SimpleBlocDelegate();
+  runApp(App());
+}
+
+class App extends StatefulWidget {
+  @override
+  State<App> createState() => _AppState();
+}
+
+class _AppState extends State<App> {
+  AuthenticationBloc _authenticationBloc;
+  FirebaseAuthService _firebaseAuthService;
+
+  @override
+  void initState() {
+    _firebaseAuthService = FirebaseAuthService();
+    _authenticationBloc = AuthenticationBloc(firebaseAuthService:_firebaseAuthService);
+    _authenticationBloc.dispatch(AppStarted());
+    super.initState();
```

Comment:
I think `super.initState()` should be the first thing called.

Reply:
My bad, I have changed it now.